ext/nnstreamer/android_source/gstamcsrc_looper.h

```diff
@@ -0,0 +1,83 @@
+/**
+ * GStreamer Android MediaCodec (AMC) Source Looper
+ * Copyright (C) 2019 Samsung Electronics Co., Ltd. All rights reserved.
+ * Copyright (C) 2019 Dongju Chae <dongju.chae@samsung.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ */
+/**
+ * @file	  gstamcsrc_looper.h
+ * @date	  19 May 2019
+ * @brief	  GStreamer source element for Android MediaCodec (AMC)
+ * @see		  http://github.com/nnsuite/nnstreamer
+ * @author	Dongju Chae <dongju.chae@samsung.com>
+ * @bug		  No known bugs except for NYI items
+ */
```

Comment:
Please add more information about looper: What it is & Why it is needed. (It appears that looper is a "thread" for source-android)

Reply:
OK. I'll update this.

Reply:
Great!